app/scripts/web/blueprints/lecture.py

```
@@ -0,0 +1,305 @@
+    chunks_path = DEFAULT_LECTURE_CHUNKS
+    if not chunks_path.exists():
+        chunks_path.parent.mkdir(parents=True, exist_ok=True)
+        chunks_path.write_text("", encoding="utf-8")
+
+    existing_ids: set[str] = set()
+    for line in chunks_path.read_text(encoding="utf-8").splitlines():
+        if line.strip():
+            try:
+                existing_ids.add(json.loads(line)["id"])
+            except Exception:
+                pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)